src/pybind/mgr/cephadm/tests/test_services.py

```diff
@@ -784,58 +784,46 @@ def test_prometheus_config_security_disabled(self, _run_cephadm, cephadm_module:
 
                 scrape_configs:
                   - job_name: 'ceph'
-                    honor_labels: true
```

Comment:
are these changes mandatory? because we are just changing the order ... if no plz keep the previous order